radar-commons-android/src/main/java/org/radarbase/android/util/PermissionHandler.kt

```diff
@@ -139,13 +139,14 @@ open class PermissionHandler(
                 dialog.cancel()
                 requestPermissions()
             }
+            show()
```

Comment:
the `alertDialog` already does `show()`.

Reply:
No. If I remove it, it doesn't show.